specs/_template/01_main.rst

```diff
@@ -225,29 +161,27 @@ Postconditions
 * Some other postcondition
 
 
-4.3. Remember loading state scenario
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+<Remember to specify loading state scenario>
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Inherits from the `4.1. Happy Case`_
+Inherits from the `Happy Case`_
 
 Steps
 +++++
 
 2. Something is loading
 
-4.5. Remember error situations
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+<Remember to specirfy error situations>
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Inherits from the `4.1. Happy Case`_
+Inherits from the `Happy Case`_
 
 Steps
 +++++
 
 3. Error happens. Display error to the user
 
 
-.. _`iOS`: 02_user_interface_ios.rst
-.. _`Android`: 02_user_interface_android.rst
-.. _`Chrome`: 02_user_interface_chrome.rst
+.. _`User Interface`: 02_user_interface.rst
 .. _`External Communication`: 03_external_communication.rst
```

Comment:
We removed that and the next should start with 03 (at least that is how your files look right now 😉 )

Reply:
forgot to update this link